frontend/tests/e2e/testCases/studentAdmin/institute/deleteSchoolPrincipal.js

```diff
@@ -0,0 +1,12 @@
+const { teardownSchoolPrincipal } = require('../../../helpers/schoolUtils');
+const log = require("npmlog");
+
+teardownSchoolPrincipal().then(async (data) => {
```

Comment:
Similar to the comment below. I think we need to delete the school and the contact. 

So I think this would be like teardownSchoolPrincipal and deleteSchool?

Reply:
Resolved, we are keeping the test School in place for testing.